CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## [2.1.0]
```

Comment:
Please could you update the version property in the `package.json` file to `2.1.0`?